### PR TITLE
session: upgrade tidb_max_chunk_size to new low bound value

### PR DESCRIPTION
**Pull request by lysu**
### What problem does this PR solve? 

old tidb(before 084aa5a03093a275b4a1b28ee50ff2dee2c1771a) with `tidb_max_chunk_size` < 32 configuration could not start when update to after 084aa5a03093a275b4a1b28ee50ff2dee2c1771a.

### What is changed and how it works?

upgrade tidb_max_chunk_size to new low bound value if the previous value is small than 32 in bootstrap.

### Check List 

Tests 

 - Manual test (add detailed scripts or steps below)

```
> set global tidb_max_chunk_size = 1; //in old tidb
// shutdown and start new version tidb without any error.
> select @@tidb_max_chunk_size;  // got 32
```

Code changes

 - update config.

Side effects

 - N/A

Related changes

 - N/A

---
This change is [Reviewable](https://reviewable.io/reviews/pingcap/tidb/8969)

### CODE REVIEW

File: session/bootstrap.go [hunk 1]
```diff
@@ -670,6 +675,13 @@ func upgradeToVer24(s Session) {
 	writeSystemTZ(s)
 }
 
+// upgradeToVer25 updates tidb_max_chunk_size to new low bound value 32 if previous value is small than 32.
+func upgradeToVer25(s Session) {
+	sql := fmt.Sprintf("UPDATE HIGH_PRIORITY %s.%s SET VARIABLE_VALUE = '32' WHERE VARIABLE_NAME = '%s' AND VARIABLE_VALUE < 32",
```
Comment (zz-jason) [inline session/bootstrap.go]: maybe it's better to use "DefTiDBMaxChunkSize" to replace the magic number "32"?

**Comment by eurekaka** [comment]
@zhouqiang-cl please take a look at the CI failure?

**Pull request merged**